SoundUtils/Sounds.cs

```diff
@@ -14,6 +14,7 @@ public void AttacksBeep()
             Console.Beep(500, 100);
             Console.Beep(600, 100);
             Console.Beep(400, 100);
+            Console.ReadLine();
```

Comment:
This is going to stop the program. We don't want the sound library to do stops like this.